api/typeDefs/user.js

```diff
@@ -104,7 +104,7 @@ export default gql`
     github: Boolean!
     twitter: Boolean!
     email: String
-    apiKey: String
+    apiKey: Boolean!
```

Comment:
What is the purpose of this boolean, just to indicate whether you've configured an API key?

Reply:
Yes. With this, the frontend can tell if the button is disabled because API key generation was not enabled or because there already is an API key which they explicitly have to delete first.